tools/archetype/binding/src/main/resources/archetype-resources/src/main/java/internal/__bindingIdCamelCase__Configuration.java

```diff
@@ -15,17 +15,20 @@
  */
 package ${package}.internal;
 
+import org.eclipse.jdt.annotation.NonNullByDefault;
+
 /**
  * The {@link ${bindingIdCamelCase}Configuration} class contains fields mapping thing configuration parameters.
  *
  * @author ${author} - Initial contribution
  */
+@NonNullByDefault
 public class ${bindingIdCamelCase}Configuration {
 
     /**
      * Sample configuration parameters. Replace with your own.
      */
-    public String hostname;
-    public String password;
+    public String hostname = "";
+    public String password = "";
     public int refreshInterval;
```

Comment:
Please set it to 600 by default to be compliant with the XML.

Reply:
I left the default value intentionally to avoid any redundancy between here and the XML. But I'm okay with it to add it.

Reply:
In practice, I always take care to set the initial value to match the default value in the XML, and I even recommend developers to do it when I review PR. So I think it would be better to show in the skeleton this good practice.

Reply:
I changed it, although I'm a big fan of avoiding redundancies.